### ISSUE TITLE
Resize the keyboard vertically for optimal fit

### ISSUE THREAD

**Issue by firelionml** [akalongman/php-telegram-bot#351]
Hi!

It is impossible to change the vertical keyboard.

```
    public function execute()
    {
        $keyboard = new Keyboard([
            ['text' =>'Hello'],
        ]);

        //Return a random keyboard.
        $keyboard
            ->setResizeKeyboard(true)
            ->setOneTimeKeyboard(true)
            ->setSelective(false);

        $chat_id = $this->getMessage()->getChat()->getId();
        $data = [
            'chat_id'      => $chat_id,
            'text'         => 'Text',
            'reply_markup' => $keyboard,
        ];

        return Request::sendMessage($data);
    }
```
What I do not so?
Screenshot https://www.dropbox.com/s/mmbwfwwpt3i6i46/2016-12-18_223313.png?dl=0

**Comment by noplanman** [comment]
This bug is fixed with ae6c3ef2066abeec233fac951cc0b6fbef56c5c0 
Could you try by manually adding the change? Just to make sure that it was that.

**Comment by firelionml** [comment]
Thanks It works)
